### DISCUSSION TITLE
Welcome, New Maintainer!

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#600] [Announcements]
Dear friends of ML!

I'm both sad and delighted - sad to step down as maintainer of this repo but delighted to welcome Carlotta Castelluccio as your new maintainer. Carlotta is an Education Cloud Advocate who will be able to shepherd you through the ins and outs of ML beautifully. Thank you all for making this repo a special, friendly place where we can all learn together. And Au Revoir!

**Comment by carlotta94c** [comment]
Thank you @jlooper, you'll be definetely missed. And hello everyone, looking forward to collaborate with you all!
